src/testFixtures/java/uk/gov/di/ipv/cri/common/library/config/EnvironConfig.java

```diff
@@ -0,0 +1,20 @@
+package uk.gov.di.ipv.cri.common.library.config;
+
+import java.util.Optional;
+
+public final class EnvironConfig {
```

Comment:
How come we have called this `EnvironConfig ` over `EnvironmentConfig` ?

Reply:
Me trying to balance what sensible enough and sprawling java code. Spotlessless will start to make a single line of code sprawl multiple line

Reply:
`EnvConfig` or just `Env`?  Perhaps Env since this isn't a config file